src/current/_includes/releases/v23.1/v23.1.23.md

```diff
@@ -0,0 +1,80 @@
+## v23.1.23
+
+Release Date: June 20, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-23-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed a bug that was present since v22.2 where [changefeeds]({% link v23.1/change-data-capture-overview.md %}) with long-running [initial scans]({% link v23.1/create-changefeed.md %}#initial-scan) might incorrectly restore checkpoint job progress and drop events during [changefeed restarts]({% link v23.1/changefeed-messages.md %}#duplicate-messages) due to transient errors or node restarts. The bug was most likely to occur in clusters with the following contributing factors:
+    - The cluster settings `changefeed.frontier_checkpoint_frequency` and `changefeed.frontier_highwater_lag_checkpoint_threshold` were set low, which resulted in the initial scan taking many multiples of the configured frequency to complete.
+    - There were multiple target tables with significant differences in row counts in one changefeed.
+    - The changefeed target tables were large with many ranges.
+    - The initial scan took a long time to complete (an hour or longer). [#123970][#123970] [#124759][#124759]
+- [Changefeeds]({% link v23.1/change-data-capture-overview.md %}) now default to evenly distributing their work across all replicas, including followers, regardless of leaseholder placement. To disable this behavior, set the [cluster setting]({% link v23.1/cluster-settings.md %}) `changefeed.balanced_distribution.enabled` to `false`. If disabled, changefeed planning reverts to its previous behavior for distributing work. [#124930][#124930]
+
+<h3 id="v23-1-23-sql-language-changes">SQL language changes</h3>
+
+- Introduced three new [cluster settings]({% link v23.1/cluster-settings.md %}) for controlling table statistics forecasting:
+    - [`sql.stats.forecasts.min_observations`]({% link v23.1/cluster-settings.md %}) is the minimum number of observed statistics required to produce a forecast.
+    - [`sql.stats.forecasts.min_goodness_of_fit`]({% link v23.1/cluster-settings.md %}) is the minimum R² (goodness of fit) measurement required from all predictive models to use a forecast.
+    - [`sql.stats.forecasts.max_decrease`]({% link v23.1/cluster-settings.md %}) is the most a prediction can decrease, expressed as the minimum ratio of the prediction to the lowest prior observation. [#124076][#124076]
+- Added a new [session setting]({% link v23.1/session-variables.md %}) `optimizer_use_improved_multi_column_selectivity_estimate`, which if enabled, causes the [optimizer]({% link v23.1/cost-based-optimizer.md %}) to use an improved selectivity estimate for multi-column predicates. This setting will default to `true` on v24.2 and later, and `false` on earlier versions. [#124076][#124076]
```

Comment:
Is this really referencing v24.2? Just double-checking that we're ok calling out a version that isn't even released yet.

Reply:
I copied this note from [v24.1.0-rc.1](https://www.cockroachlabs.com/docs/releases/v24.1.html#v24-1-0-rc-1-sql-language-changes) and [v23.2.6](https://www.cockroachlabs.com/docs/releases/v23.2.html#v23-2-6-sql-language-changes).

So I will keep the reference to v24.2.